src/vs/workbench/contrib/notebook/browser/viewParts/notebookEditorStickyScroll.ts

```diff
@@ -208,6 +245,18 @@ export class NotebookStickyScroll extends Disposable {
 		this._contextMenuService.showContextMenu({
 			menuId: MenuId.NotebookStickyScrollContext,
 			getAnchor: () => event,
+			actionRunner: {
+				run: async (action) => {
+					if (action.id === 'notebook.action.runInSection') {
+						this.commandService.executeCommand(action.id, event.target, this.currentStickyLines, this.notebookEditor);
+					} else {
+						this.commandService.executeCommand(action.id);
+					}
+				},
+				onDidRun: new Emitter<IRunEvent>().event,
+				onWillRun: new Emitter<IRunEvent>().event,
```

Comment:
from what I read https://github.com/microsoft/vscode/blob/673a1aeca1ed358222c25de98570f23060997b49/src/vs/platform/contextview/browser/contextMenuHandler.ts#L85-L86 , the context menu dismisses when run* events are triggered so we can't just have a dummy event emitter here.

Reply:
I'll go ahead and implement a more thorough actionRunner then 👍